devlog/_plan/260902_cursor_bundle_effort_table/050_wp5_claude_id_normalizer.md

```
@@ -0,0 +1,200 @@
+    if (!n || !CURSOR_CAPABILITIES[n.canonicalBaseId]) continue;
+    if (!next.has(n.canonicalBaseId)) next.set(n.canonicalBaseId, { sourceBaseId: n.sourceBaseId, spelling: n.spelling });
+  }
+  liveCursorClaudeWireIdentities = next; // replaced, never merged: a renamed model must not keep a stale spelling
+}
+export function liveCursorClaudeWireIdentitiesForTests(): ReadonlyMap<string, CursorLiveClaudeWireIdentity> { return liveCursorClaudeWireIdentities; }
+export function resetLiveCursorClaudeWireIdentitiesForTests(): void { liveCursorClaudeWireIdentities = new Map(); }
+```
+4. `composeWireId(baseId, kind, effort, claudeIdentity?)` (line ~545): when `claudeIdentity` is
+   given, return `composeCursorClaudeWireId(claudeIdentity, { thinking, fast, effort, bareThinking: spec.order === "bare" })`;
+   the non-Claude body is unchanged.
+5. `resolveCursorSelection`: `const claudeIdentity = liveCursorClaudeWireIdentities.get(parsed.baseId) ?? (requestedClaude ? { sourceBaseId, spelling } : undefined)`
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 2; Actual: 5; Style: 1/2/3

(MD029, ol-prefix)